src/wallet/rpc/wallet.cpp

```diff
@@ -220,7 +220,7 @@ static RPCHelpMan loadwallet()
                 "\nNote that all wallet command-line options used when starting bitcoind will be"
                 "\napplied to the new wallet.\n",
                 {
-                    {"filename", RPCArg::Type::STR, RPCArg::Optional::NO, "The wallet directory or .dat file."},
+                    {"filename", RPCArg::Type::STR, RPCArg::Optional::NO, "Filename relative to the wallet directory."},
```

Comment:
Thanks for the review. Is this description better? 
``` 
You may pass in the absolute path of your wallet directory or .dat file. Otherwise, the interpreted path will be relative to the default wallet directory. 
```

Reply:
I can also add examples that load wallet using an absolute path.

Reply:
Based on https://github.com/bitcoin/bitcoin/blob/master/doc/files.md#sqlite-database-based-wallets, it appears `.dat` files can be passed to sqlite wallets as well.

Reply:
Added additional examples